crew

```diff
@@ -407,6 +407,9 @@ end
 def build_and_preconfigure (target_dir)
   Dir.chdir target_dir do
     puts "Building from source, this may take a while..."
+    if ARCH == "x86_64"
+      @pkg.remove_la_files
```

Comment:
Please remove the logic from package.rb and put here instead.  You don't even need to define a function out of it.

Reply:
Where are the instructions to execute `make distclean`?  Isn't that part of the process?

Reply:
Don't need it, since we don't perform any "make" process before.